build.ps1

```diff
@@ -193,6 +193,9 @@ $skip_test_projects_on_windows = @("tree-sitter-dscexpression")
             }
 
             Copy-Item "*.dsc.resource.json" $target -Force -ErrorAction Ignore
+            
+            $psdscFolder = New-Item -Path $target -Name 'psDscAdapter' -Force -ItemType Directory -ErrorAction Ignore
```

Comment:
Instead of this, you should use a `copy_files.txt` file which just lists the files to be copied to the release folder

Reply:
fixed